recipe/meta.yaml

```diff
@@ -9,35 +8,36 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: faa507115731742e4eb0fbfec7be41310efcc46523afc188440b2d26ed48d361
 
 build:
   number: 0
+  # Windows skip is due to snowflake only build of tiledb 2.26.2
+  skip: True  # [py<39 or s390x or win]
+  ignore_run_exports:
```

Comment:
you don't need this to change the pinning on numpy

Reply:
ok in that case you should remove it from run as well

Reply:
ok, it's weird that it's using numpy at compile time but not runtime. Also given the listing in [pyproject.toml](https://github.com/TileDB-Inc/TileDB-Py/blob/0.32.5/pyproject.toml#L40) @cbouss @JeanChristopheMorinPerso do either of you know what's going on?

Reply:
Can it be because the runtime numpy has a lower requirements?
Also note: I've added it to runtime because that's what upstream states, but testing shows it's not necessary.

Reply:
> Can it be because the runtime numpy has a lower requirements?

I don't think you'd need to ignore_run_exports, just override it in the run section.

Anyway, I think it's ok functionally, just got the feeling there was something I didn't understand.